bundles/org.openhab.persistence.dynamodb/pom.xml

```diff
@@ -15,118 +15,293 @@
   <name>openHAB Add-ons :: Bundles :: Persistence Service :: DynamoDB</name>
 
   <properties>
-    <dep.noembedding>jackson-core,jackson-annotations,jackson-databind,jackson-dataformat-cbor</dep.noembedding>
-    <bnd.importpackage>!com.amazonaws.*,!org.joda.convert.*,!com.sun.org.apache.xpath.*,!kotlin,!org.apache.log.*,!org.bouncycastle.*,!org.apache.avalon.*</bnd.importpackage>
+    <!-- Avoid declaring OSGI-imports for packages that are part of embedded/compiled dependencies, declared below under
+      <dependencies> -->
+    <bnd.importpackage>!com.amazonaws.*,!com.sun.org.apache.xpath.*,!kotlin,!org.apache.log.*,!org.bouncycastle.*,!org.joda.convert.*,!scala.util.*,!software.amazon.*</bnd.importpackage>
```

Comment:
No we need to exclude `org.slf4j.impl` in `bnd.importpackage`? Some of the below `<dependencies>` dependencies depend on `org.slf4j.api -- should we exclude that dependency using `<exclusions>` similar to how it has been done in some other addons?